apps/web/src/studio/hooks/useIsStudio.ts

```diff
@@ -0,0 +1,9 @@
+import { useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+import { ROUTES } from '../../constants/routes';
+
+export const useIsStudio = () => {
+  const { pathname } = useLocation();
+
+  return useMemo(() => pathname.startsWith(ROUTES.STUDIO), [pathname]);
+};
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** We already have a util for it, `isStudioRoute`‏

Reply:
I know the problem for me with the util is that path can contain /studio and do not need to start with it so if a workflow is named studio it will return true and can be problematic. Should we change it instead?